pkgs/applications/misc/solaar/default.nix

```diff
@@ -18,6 +18,11 @@ pythonPackages.buildPythonApplication rec {
     wrapProgram "$out/bin/solaar-cli" \
       --prefix PYTHONPATH : "$PYTHONPATH" \
       --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH"
+
+    mkdir -p $out/lib/udev/rules.d
+    for f in rules.d/*.rules; do
+      cp "$f" $out/lib/udev/rules.d/
```

Comment:
what about `cp -r rules.d $out/lib/udev` ?

Reply:
there's an install.sh in the directory :)

Reply:
ah, then it would be `cp rules.d/*.rules $out/lib/udev/rules.d/`